Lib/tkinter/__init__.py

```diff
@@ -4877,26 +4893,17 @@ def sash_place(self, index, x, y):
         return self.sash("place", index, x, y)
 
     def panecget(self, child, option):
-        """Query a management option for window.
-
-        Option may be any value allowed by the paneconfigure subcommand
-        """
+        """Return the value of option for window."""
```

Comment:
Not sure about the parameter naming here. The method only has a `child` param... perhaps something like:

```suggestion
        """Return the value of option for a the window child."""
```

Understandability here would definitely benefit from adding formatting, as long as it would render in the docs: `*child*`